doc/releases.md

```diff
@@ -14,41 +19,46 @@ CDI adheres to the [semantic version definitions](https://semver.org/) format of
 
 ### Releasing a New Version
 
- The version number is tracked in several files in CDI as well as through a git tag.  To reduce the chance of human error, a help script is used to change the version in all known locations.
+Release branches are used to isolate a stable version of CDI.  Git tags are used within these release branches to track incrementing of Minor and Patch versions.  When a Major version is incremented, a new stable branch should be created corresponding to the release.
 
-     DO NOT EDIT ANY VERSION STRINGS IN CDI!!
+- Release branches should adhere to the `release-v#.#.#` pattern.
 
-1. Set the new release version
+- Tags should adhere to the `v#.#.#(-alpha.#)` pattern.
```

Comment:
Better without the `*` but the parenthesis still confuse me. What do they mean in this context?

Reply:
As in `The dog(s) is (are) playing in the yard.`